src/Client/Server/Server.php

```diff
@@ -102,7 +102,10 @@ public function getAuthorizationUrl($temporaryIdentifier)
 
         $parameters = array('oauth_token' => $temporaryIdentifier);
 
-        return $this->urlAuthorization().'?'.http_build_query($parameters);
+        $url = $this->urlAuthorization();
+        $query_string = http_build_query($parameters);
```

Comment:
Could you please switch to camelCasing for the three occurrences and docblocks of this variable?